packages/dev/core/src/Meshes/abstractMesh.ts

```diff
@@ -1458,6 +1458,19 @@ export class AbstractMesh extends TransformNode implements IDisposable, ICullabl
         this._updateBoundingInfo();
     }
 
+    /**
+     * Computes a fresh bounding info from the mesh positions and cached world matrix.
+     * @returns the bounding info obtained from the mesh positions and cached world matrix, or null if the mesh has no position data.
+     */
+    public getBoundingInfoFromVerticesData() {

```

Comment:
Why do we need a new one ? instead of updating and cloning ?

Reply:
ping @carolhmj

Reply:
Oh, that's because when a mesh has thin instances, the bounding box doesn't represent a single instance only, which is what we need in physics @sebavan

Reply:
makes sense but in this case should it not be instead a helper function or directly within the physics API ?

This would help not growing mesh for none physics users (agree it is really small here) and prevent the confusion I had

Reply:
Also, are the calls to `getBoundingInfoFromVerticesData` on the hot path, i.e. can they be called multiple times per frame? If so, we should probably find a way to cache the result (and only update the cached result when necessary)...

Maybe we should add something like `getOwnBoundingInfo` to mesh, which would return the same bounding info as `getBoundingInfo` when there are no thin instances, but a bounding info that does not include thin instances when there are?

Reply:
Yeah I was thinking it might be useful to cache the "original" bounding info of the mesh somewhere, as this is going to be needed for the PhysicsHelper too...

Reply:
@CedricGuillemet do you want to remove the `getBoundingInfoFromVerticesData` from the PR, and I add this mechanism of a separate bounding box for instanced meshes in a separate PR for core?

Reply:
I let you do the change/tests in private repo and when it's ready, I'll update this PR.

Reply:
@carolhmj is working on it this week, with all the new changes.

Reply:
No and we are doing the last stretch of v2. I'll update this PR when it's done. I'm setting it to draft until it's ready (ETA tomorrow or Wednesday)